heron/tools/tracker/src/python/main.py

```diff
@@ -125,6 +125,8 @@ def cli(
 
   log_level = logging.DEBUG if verbose else logging.INFO
   log.configure(log_level)
+  global Log
+  Log = log.Log
```

Comment:
I assume the purpose of this line was to update the globally defined `Log`? If this is the case, we need the `Log` variable to be defined as `global` as it is now.